src/driver/AmdCompiler.cpp

```diff
@@ -79,6 +81,9 @@ using namespace llvm::object;
 using namespace clang;
 using namespace clang::driver;
 
+static cl::opt<bool>
+Verbose("v", cl::ZeroOrMore, cl::desc("Print information about actions taken"), cl::init(false));
```

Comment:
The name of the option shall be changed. All options from linked in llvm have the same namespace, so -v is too generic.